app/src/main/java/net/squanchy/service/firestore/injection/FirestoreModule.kt

```diff
@@ -1,20 +1,31 @@
 package net.squanchy.service.firestore.injection
 
+import com.google.firebase.auth.FirebaseAuth
 import com.google.firebase.firestore.FirebaseFirestore
 import dagger.Module
 import dagger.Provides
 import net.squanchy.injection.ApplicationLifecycle
+import net.squanchy.service.firestore.FirebaseAuthService
 import net.squanchy.service.firestore.FirestoreDbService
 
 @Module
 class FirestoreModule {
```

Comment:
probably the best thing is to have everything inside a `firebase` package (which is the 3rd party offering the services) and then we'll have `FirestoreDbService` (which is the particular firebase service we're using)